cli/cmd/top.go

```diff
@@ -630,6 +654,25 @@ func (t *topTable) renderHeaders() {
 	}
 }
 
+func (t *topTable) tableWidthCalc() int {
+	tablewidth := 0
+	for i, col := range t.columns {
+		t.columns[i].width = runewidth.StringWidth(col.header)
```

Comment:
We always call `tableWidthCalc` after calling `adjustColumnWidths` so I don't think we need to adjust the column widths here and can assume that the column widths are already correct.

Reply:
Done :)